app/manage/lib/core/screens/user_profile_screen.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:manage/core/model/user_profile_model.dart';
+
+
```

Comment:
This class should be named as ...Screen, at least in outer most class.
You didn't connect this screen to proper places so there is no way to reach this screen, but you should have implemented this.